GOOLCodeGen/GlassBR/Main.hs

```diff
@@ -2,7 +2,14 @@ module Main where
 
 import System.Directory
 
-import Language.Drasil.Code
+import Language.Drasil.Code.Imperative.New (Label)
```

Comment:
BTW. the `New` names can stay for the first PR, then be removed in a second pass.